.editorconfig

```diff
@@ -18,3 +18,8 @@ trim_trailing_whitespace = true
 ij_kotlin_code_style_defaults = KOTLIN_OFFICIAL
 ij_kotlin_name_count_to_use_star_import = 999
 ij_kotlin_name_count_to_use_star_import_for_members = 999
+
+[renovate.json]
```

Comment:
Guess this could be *.json

Reply:
Oh right, now that I changed it. Will fix.